packages/metrics/test/unit/aggregate.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { expect } from 'chai';
+import { MutationTestResult } from 'mutation-testing-report-schema/api';
+import { aggregateResultsByModule } from '../../src';
+import { createFileResult, createMutationTestResult, createTestFile } from '../helpers/factories';
+
+describe(aggregateResultsByModule.name, () => {
+  it('should result in an empty report when an empty object is provided', () => {
+    const expectedReport: MutationTestResult = {
+      files: {},
+      schemaVersion: '1.7',
+      thresholds: { high: 80, low: 60 },
+      config: {},
+      projectRoot: undefined,
+    };
+    expect(aggregateResultsByModule(Object.create(null))).deep.eq(expectedReport);
```

Comment:
What if an empty object is passed to the function? This `Object.create` thing makes me worry it gives unexpected results

Reply:
`Object.create(null)` is the emptiest object there is. Even more empty than `{}`, since it won't even have `Object.prototype` as its prototype

Reply:
Does `{}` give a different result?

Reply:
Of course 😌. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create#custom_and_null_objects

Reply:
Then that should probably also have a test, right? `{}` seems a lot more common 🤔

Reply:
Good point. Added the test